org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/AST.java

```diff
@@ -509,6 +524,11 @@ public final class AST {
 	 * deprecation warnings once JLS21 is deprecated
 	 */
 	static final int JLS22_INTERNAL = JLS22;
+	/**
+	 * Internal synonym for {@link #JLS22}. Use to alleviate
+	 * deprecation warnings once JLS21 is deprecated
```

Comment:
Should be `JLS23` and `JLS22` in the doc.

Reply:
Done